firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -389,6 +389,7 @@ class BrowserViewController: UIViewController,
         tabManager.preserveTabs()
         // TODO: [FXIOS-7856] Some additional updates for telemetry forthcoming, once iPad multi-window is enabled.
         TabsQuantityTelemetry.trackTabsQuantity(tabManager: tabManager)
+        SearchBarSettingsViewModel.recordLocationTelemetry(for: isBottomSearchBar ? .bottom : .top)
```

Comment:
Won't this send too many events? 🤔

Reply:
R+ in general but could we get a clarity if we are okay sending many such events as soon as we go to background from glean team :)

Reply:
basically the current implementation isn't sending _enough_ events so only 87% of users send this event in a given day, sending this event here would make sure that no matter what, when the app loses focus it will send this info. The alternative would be sending this event in another method of the view lifecycle, but I thought that would send even more events because viewWillAppear/DidAppear would get called even more frequently than the background event for the same outcome. Is there anyone in the glean team I can discuss this issue with or should I send a message to the glean channel about it?